### PR TITLE
chore: bump package registry to staging

### LINKED ISSUE #253: [Ingest Manager] Wrong package versions, need to update to using a :distribution image (tested w 7.10.0)
I ran the test recently with a 7.10 variable citation as below:

export DEVELOPER_MODE=true && ELASTIC_AGENT_VERSION=7.10.0-SNAPSHOT STACK_VERSION=7.10.0-SNAPSHOT godog -t @enroll

This pulled what seems to be the 7.8.x list of packages, just the original 9 of them that we had then.

I need help reviewing how the branches in e2e-testing are set up and what we might be able to do here... at a minimum, we could make sure the Registry image we use is always refreshed with latest Integration packages from the storage repo it points to (snapshot, staging, or production).

Its a bit of research to figure out, I don't know yet whats up.  Maybe Ruflin can help or SKH who has worked with getting the registry docker container in use in Kibana side.

screenshots:

Endpoint on at 0.8.0 which is old
Screen Shot 2020-08-28 at 10 26 45 AM

base 9 integrations shown:
Screen Shot 2020-08-28 at 10 26 29 AM

### PR DESCRIPTION
**Pull request by mdelapenya**
## What does this PR do?
This PR bumps the package registry to `staging`. After the change, we noticed the title for the "Elastic Endpoint" integration changed to "Elastic Endpoint Security", so we renamed the feature files and other usages in consequence.

## Why is it important?
The staging tag is updated more frequently and, for master branch, we want to use latest code.

About the usage of the title instead of the name, we are using the title because the feature files have to be super readable, and the title is more readable than the name, which is usually an identifier that could bring less value from business side.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have run the Unit tests for the CLI, and they are passing locally
- [x] I have run the End-2-End tests for the suite I'm working on, and they are passing locally
- [ ] I have noticed new Go dependencies (run `make notice` in the proper directory)

## How to test this PR locally
To run the endpoint integration scenarios:

```shell
$ cd e2e/_suites/ingest-manager
$ OP_LOG_LEVEL=TRACE DEVELOPER_MODE=true godog -t "agent_endpoint_integration"
```

## Related issues

- Closes #253

## Use cases

```gherkin
Given the master branch
When we run the tests
Then the current development version of the registry is used

Given a maintenance branch (7.x.x)
When we run the tests
Then the released version ffor that branch is used
```

### CODE REVIEW

**Review by cachedout — APPROVED** [review]

File: cli/config/compose/profiles/ingest-manager/docker-compose.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ services:
     volumes:
       - ${kibanaConfigPath}:/usr/share/kibana/config/kibana.yml
   package-registry:
-    image: docker.elastic.co/package-registry/package-registry:41c150c8020efc53ab16e3bba774c62a419b51ea
```
Comment (ruflin) [inline cli/config/compose/profiles/ingest-manager/docker-compose.yml:38]: I'm tempted to say we should use here `distribution:snapshot` and use `distribution:staging` for the 7.x e2e tests.

Comment (mdelapenya) [reply cli/config/compose/profiles/ingest-manager/docker-compose.yml:38]: I we have this tag, I'm OK with it.

For the 7.x branch, we do not have that branch, only 7.9.x and eventually 7.10.x. Should we have a 7.x branch?

Comment (ruflin) [reply cli/config/compose/profiles/ingest-manager/docker-compose.yml:38]: As we have 7.* and 8.* branches I assume we are aligned with the release branches. So I think yes. Then it becomes self explaining on what monitors which part.

Comment (mdelapenya) [reply cli/config/compose/profiles/ingest-manager/docker-compose.yml:38]: Good. We'd need then to define which versions for the stack are needed, as there is no https://container-library.elastic.co/r/elasticsearch/elasticsearch:7.x image

Comment (mdelapenya) [reply cli/config/compose/profiles/ingest-manager/docker-compose.yml:38]: We could use this issue for reference: https://github.com/elastic/e2e-testing/issues/274

Comment (ruflin) [reply cli/config/compose/profiles/ingest-manager/docker-compose.yml:38]: There actually is one: `docker.elastic.co/elasticsearch/elasticsearch:7.x-SNAPSHOT`

**Pull request merged**